test/fixtures/mson/enum-variants.json

```diff
@@ -128,15 +139,15 @@
                             "content": [
                               {
                                 "element": "string",
-                                "content": "S"
+                                "content": "L"
                               },
                               {
                                 "element": "string",
-                                "content": "M"
+                                "content": "S"
                               },
                               {
                                 "element": "string",
-                                "content": "L"
+                                "content": "M"
```

Comment:
This matches any `string`. Are you missing `fixed` attribute on the `string` elements above?